_build/builder/BuildDocs.py

```diff
@@ -116,22 +123,24 @@ def Setup(self):
             self.activated = bool(self.buildSetup.args.sync)
 
     def DoTask(self):
-        Prepare()
-        sphinx.build_main([
-            None,
-            "-a",
-            "-b", "html",
-            "-E",
-            "-N",
-            "-P",
-            "-D", "release=%s" % self.buildSetup.appVersion,
-            "-d", join(self.buildSetup.tmpDir, ".doctree"),
-            DOCS_SOURCE_DIR,
-            join(self.buildSetup.websiteDir, "docs"),
-        ])
-
-
-def BuildClsDocs(clsNames):
+        WritePluginList(join(self.buildSetup.docsDir, "pluginlist.rst"))
+        Prepare(self.buildSetup.docsDir)
+        sphinx.build_main(
+            SPHINX_COMMON +
+            [
+                "-b", "html",
+                "-D", "version=%s" % self.buildSetup.appVersion,
+
+
+                "-D", "release=%s" % self.buildSetup.appVersion,
+                "-d", join(self.buildSetup.tmpDir, ".doctree"),
+                self.buildSetup.docsDir,
+                join(self.buildSetup.websiteDir, "docs"),
+            ]
+        )
+
+
+def BuildClsDocs(clsNames, doc_src_dir):
```

Comment:
I am thinking we should not include a list of plugins in the docs. as these are external pieces of code that technically speaking not part of the core of EG and are supposed to be maintained by a 3rd party. people think that because it is included in the core that they are maintained by us. Which on a side not we should change the authors to any of the ones that are maintained or written by the core team. to "The EventGhost Team" This can be changed in the future I guess. I do not even know if the list is correct to the plugins that are shipped with EG. I would have to check.